### ISSUE TITLE
JAWS show incorrect form fields in Internet Explorer 11

### ISSUE THREAD

**Issue by maksimsech** [FreedomScientific/VFO-standards-support#378]
### Summary

> Table have custom actions in column headers and grid cells.

### Expected result

> Chrome shows: Questions table.

### Actual result

> IE shows: QuestionsSort by question nameActionsInformation about Jaws questionEdit Jaws questionDelete Jaws question table.

### Example
> Example [https://jsbin.com/simuxuyaqa/edit?html,output](url)

#### JAWS version and build number

> JAWS 2019.1912.1.400
 
#### Operating System and version

> Windows 10 1903

#### Browser and version:

> Internet Explorer 11

**Comment by menovak** [comment]
Hi,

I'm not sure I understand your question?  I loaded up your JSBIN into Codepen, as that is what I use and recorded the output from JAWS when navigating the table/grid using the standard JAWS table navigation keys.  I've included what I heard (as output from JAWS speech history) below.  

I did see that there is some issues with your markup/code.  For example, using an anchor without any href and then applying aria-label, may not be announced as the anchor without the href since the anchor alone doesn't have a role of 'link'.  Likewise, in your column header, the visible text becomes the column header name, not the anchor without any href which includes aria-label.  But perhaps I am missing something.....

Regards,
Mark

// output
// JAWS 2019  - get to table using 't' key in both cases

// IE 11

grid with 2 columns and 2 rows
Questions
Column 1, Row 1
Question name
Actions 
Column 2
Edit
Delete
Row 2
Question name 
Information about Jaws question
Button 
Column 1

//Chrome, Version 80.0.3987.149 (Official Build) (64-bit)

grid with 2 columns and 2 rows
Questions
Column 1, Row 1
Question name
Actions
Column 2
Edit
Delete
Row 2
Question name 
Information about Jaws question
Button 
Column 1


// code pen
https://codepen.io/menovak/pen/GRJevgM

**Comment by stevefaulkner** [comment]
Closing this issue as wontfix as its IE only
